lib/src/ui/widgets/paginator_content/scrollable_number_content.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+import 'package:number_paginator/src/ui/widgets/buttons/paginator_button.dart';
+import 'package:number_paginator/src/ui/widgets/inherited_number_paginator.dart';
+
+class ScrollableNumberContent extends StatelessWidget {
+  final int currentPage;
+
+  const ScrollableNumberContent({
+  super.key,
+  required this.currentPage,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    var numberPages = InheritedNumberPaginator.of(context).numberPages;
+
+    return SizedBox(
+      height: 50, // Set a fixed height
```

Comment:
there should not be a fixed height here. what if someone wants to use this, but wants height 60?
the height should be somehow inferred, like for example in the `number_content.dart`